CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Added interruption of an incomplete expression when pressing
   ``Ctrl-C`` in ``cartridge enter`` command.
+- Variables ``TARANTOOL_WORKDIR``, ``TARANTOOL_PID_FILE`` and
+  ``TARANTOOL_CONSOLE_SOCK`` can be customized when packing in docker via
+  ``cartridge pack`` command.
```

Comment:
In fact, this variables aren't customized on packing, but can be specified on starting container built by `cartridge pack docker`.

Both your changes (this and below) can be placed in `Added` section since we've *added* an ability to customize smth.